apps/web/src/components/Basenames/RenewalFlow.tsx

```diff
@@ -40,6 +46,24 @@ export function RenewalFlow({ name }: RenewalFlowProps) {
     }
   }, [isOnSupportedNetwork, chain, switchToIntendedNetwork]);
 
+  const isForm = renewalStep === RenewalSteps.Form;
+  const isPending = renewalStep === RenewalSteps.Pending;
+  const isSuccess = renewalStep === RenewalSteps.Success;
+
+  // Map renewal steps to registration steps for background animations
+  const backgroundRegistrationStep = useMemo(() => {
+    switch (renewalStep) {
+      case RenewalSteps.Form:
+        return RegistrationSteps.Claim; // Globe video with gray background
```

Comment:
Ideally we create a separate abstraction that both the renewal and registration flows can use, rather than needing to map renewal steps to registration steps. Can be done later on though